e2e/02-login-flow.e2e.spec.ts

```diff
@@ -86,9 +94,12 @@ describe("Login Flow", () => {
     expect(logoutButton.isDisplayed()).toBeTruthy()
     const backButtonOnAccountScreen = await $(goBack())
     await backButtonOnAccountScreen.click()
+    await browser.pause(2000)
```

Comment:
Same here

Reply:
I couldn't find a way to use the `browser.waitUntil()` fn here. We need this pause esp for android because there is small lag when we move to the settings screen from the debug screen.

Reply:
Is there something on the settings screen you could wait until it was displayed? There is also the `waitForExist` method I believe.